includes/class-wcmp-export.php

```diff
@@ -882,15 +904,12 @@ public function get_customs_declaration($order ) {
                 if (empty($classification)) {
                     $classification = $default_hs_code;
                 }
-
                 // add item to item list
                 $items[] = compact('description', 'amount', 'weight', 'item_value', 'classification', 'country');
             }
         }
-        // Get the total weight of the package
-        $weight = (int) round($this->get_parcel_weight($order) * 1000);
 
-        return compact('weight', 'invoice', 'contents', 'items');
+        return $items;
```

Comment:
Hier krijg je een melding dat de $items misschien niet gezet is, kijk hoe je dit kan oplossen.

Reply:
added $items to functions argument

Reply:
kan je checken of het nu wel goed staat